apps/aecore/src/aec_keys.erl

```diff
@@ -302,7 +302,7 @@ hash(Bin) ->
 %% INFO: keep separate APIs and encrypt both priv & pub to protect external HDs
 %%       (there is known atack vector using master pub)
 encrypt_privkey(Password, Bin) ->
```

Comment:
Hardcoding expectation on return value of `decrypt_privkey` would help readability.

Reply:
I'm not sure what you mean (did you mean `encrypt_privkey`?). The problem is that `crypto:block_encrypt/3` crashes if the key is too short and otherwise it returns a binary.